tensorboard/plugins/scalar/BUILD

```diff
@@ -122,7 +115,6 @@ py_test(
     deps = [
         ":metadata",
         ":summary",
-        "//tensorboard:expect_numpy_installed",
```

Comment:
At a glance, it looks like `:summary_test` still imports Numpy, so let’s
keep this, too?
<https://github.com/tensorflow/tensorboard/blob/8913abc1c49236b4b1aa5f6b9720cd11049c07d8/tensorboard/plugins/scalar/summary_test.py#L26>

Reply:
Restored. Good eyes. Sorry, I should have caught this.